app/Http/Controllers/Api/Client/Servers/ScheduleTaskController.php

```diff
@@ -81,6 +87,10 @@ public function update(StoreTaskRequest $request, Server $server, Schedule $sche
             throw new NotFoundHttpException();
         }
 
+        if ($server->backup_limit === 0 && $request->action === 'backup'){
+            throw new HttpForbiddenException("A backup task cannot be created when the backup limit for a server is set to {$server->backup_limit}.");
```

Comment:
This should return a bad request response, forbidden is used for bad tokens.  Message should be changed to something like `This server is not able to create any backups` or `A backup task cannot be created because the server has a backup limit of 0` (if you leave the message the same, you can replace `{$server->backup_limit}` with `0`.

Reply:
Changed to just 0 as the message would only be shown if the limit === 0. 
as for the wording, it's from Dane...